iosxr_store_box.py

```diff
@@ -33,25 +33,8 @@
 import sys
 import os
 import getopt
-import subprocess
 import smtplib
-
-
-def run(command_string, debug=False):
-    '''
-    Execute a CLI command string and return the result.
-
-    Note this does not handle operators which need whitespace.
-
-    In those cases the raw commands will need to be used.
-
-    E.g. passing grep 'a string separated by spaces'.
-    '''
-    argv = command_string.split()
-    if debug is True:
-        print ('argv: %s' % argv)
-    result = subprocess.check_output(argv)
-    return result
+from iosxr_iso2vbox import run
 
 
 def main(argv):
```

Comment:
Ah, I did iosxrv_test.py correctly, but not this one. Yes:

US117901 - iosxrv_store.py - needs to be converted to use argparse.